db/schema.rb

```diff
@@ -164,6 +164,7 @@
 
   add_index "jobs", ["project_id"], name: "index_jobs_on_project_id", using: :btree
   add_index "jobs", ["status"], name: "index_jobs_on_status", length: {"status"=>191}, using: :btree
+  add_index "jobs", ["user_id"], name: "index_jobs_on_user_id", using: :btree
```

Comment:
It seems to always randomly change ... we can try normalizing it again but that should be a separate PR ... we had so many funky issues around that ... so I just stay with hand-picked updates ....